src/main/java/de/tum/in/www1/artemis/web/rest/ComplaintResource.java

```diff
@@ -165,8 +200,7 @@ public ComplaintResource(AuthorizationCheckService authCheckService, ExerciseSer
         if (exerciseId == null && courseId == null) {
             User callerUser = userService.getUser();
```

Comment:
Can we simply name this variable `user` please?

Reply:
Yes